benchmark/util/style-text.js

```diff
@@ -29,6 +30,8 @@ function main({ messageType, format, validateStream, n }) {
       break;
   }
 
+  process.env.NO_COLORS = noColors ? true : false;
```

Comment:
```suggestion
  process.env.NO_COLORS = noColors;
```

Reply:
+1 for the first solution no condition, keep it simple